lib/js/components/Headers/SectionHeader/SectionHeader.vue

```diff
@@ -238,6 +310,38 @@
 		}
 	}
 
+	&.-size-xx-small {
+		#{$root}__main {
+			padding: $space-xxxxxs 0;
+		}
+
+		#{$root}__titleWrapper {
+			gap: $space-xxxs;
+		}
+
+		#{$root}__header {
+			@include info-s-extensive-bold-uppercase;
+		}
+
+		#{$root}__icon {
+			color: $color-neutral-icon-weak;
```

Comment:
should the color be overwritten here?

Reply:
Yes, xx-small has weaker color by default